src/components/tooltip/Tooltip.js.flow

```diff
@@ -43,6 +43,8 @@ type Props = {
     onDismiss?: () => void,
     /** Where to position the tooltip relative to the wrapped component */
     position?: Position | CustomPosition,
+    /** True when should hide tooltip (usually when another tooltip is showing) */
+    shouldHideTooltip?: boolean,
```

Comment:
can you explain why you need this and can't use `isShown`?

Reply:
same